main.tf

```diff
@@ -1 +1,57 @@
 
+resource "azurerm_mysql_flexible_server" "this" {
+  name                         = var.server_name
+  location                     = var.location
+  resource_group_name          = var.resource_group_name
+  version                      = var.server_version
+  administrator_login          = var.administrator_login
+  administrator_password       = var.administrator_password
+  sku_name                     = var.sku_name
+  geo_redundant_backup_enabled = var.geo_redundant_backup_enabled
+  zone                         = var.zone
+
+  storage {
+    auto_grow_enabled = var.auto_grow_enabled
+    size_gb           = var.size_gb
+  }
+
+  lifecycle {
+    ignore_changes = [
+      administrator_password
```

Comment:
We usually add a comment for each entry in `ignore_changes` to justify why we ignore changes to a property, similar to Azure SQL module: https://github.com/equinor/terraform-azurerm-sql/blob/2cc09d452f83c0f1c1738082b613c34e71d30691/main.tf#L32-L33